criu/crtools.c

```diff
@@ -440,6 +440,9 @@ int main(int argc, char *argv[], char *envp[])
 "			Namespace can be specified as either pid or file path.\n"
 "			OPTIONS can be used to specify parameters for userns:\n"
 "			    user:PID,UID,GID\n"
+"  --file-validation [m]\n"
```

Comment:
Nit here:
`--file-validation METHOD`
(`[m]` in square brackets usually means it's optional)